index.js

```diff
@@ -863,6 +884,13 @@
     return f(z, this.value);
   });
 
+  //  Just#sequence :: Applicative f => Maybe (f a) ~> (b -> f b) -> f (Maybe a)
+  Just.prototype.sequence =
+    def('Just#sequence', [Function], function(of) {
+      void of;
```

Comment:
I've started doing this to make it clear that the argument isn't used. Happy to remove it.